controllers/gemini.controller.js

```diff
@@ -71,29 +84,41 @@ const getTasks = async (req, res) => {
     }
 
     const mood = getMood(scale);
-    const requestData = {
-      contents: [
-        {
-          parts: [
+    const cacheKey = `tasks_${scale}`;
```

Comment:
Same thing here, no point to cache where responses are dynamic. 
In situations where the data changes frequently or is user-specific, caching doesn't provide much benefit.
@tulu-g559